label_studio/io_storages/gcs/utils.py

```diff
@@ -30,6 +32,22 @@ class ConvertBlobTo(Enum):
         JSON_DICT = 3
         BASE64 = 4
 
+    @classmethod
+    @lru_cache(maxsize=1)
+    def get_bucket(
+        cls,
+        google_project_id: Optional[str] = None,
+        google_application_credentials: Optional[Union[str, dict]] = None,
+        bucket_name: Optional[str] = None,
+        ttl_hash: Optional[int] = None,
```

Comment:
Please make this a required argument (ie, type `int`), users shouldn't call this in a way that would cause buckets to be cached indefinitely